pkg/resourceinterpreter/customized/declarative/configmanager/manager.go

```diff
@@ -84,35 +80,42 @@ func NewInterpreterConfigManager(informer genericmanager.SingleClusterInformerMa
 
 	// In interpret command, rules are not loaded from server, so we don't start informer for it.
 	if informer != nil {
+		manager.informer = informer
 		manager.lister = informer.Lister(resourceInterpreterCustomizationsGVR)
 		configHandlers := fedinformer.NewHandlerOnEvents(
-			func(_ interface{}) { manager.updateConfiguration() },
-			func(_, _ interface{}) { manager.updateConfiguration() },
-			func(_ interface{}) { manager.updateConfiguration() })
+			func(_ interface{}) { _ = manager.updateConfiguration() },
+			func(_, _ interface{}) { _ = manager.updateConfiguration() },
+			func(_ interface{}) { _ = manager.updateConfiguration() })
 		informer.ForResource(resourceInterpreterCustomizationsGVR, configHandlers)
 	}
 
 	return manager
 }
 
-func (configManager *interpreterConfigManager) updateConfiguration() {
+func (configManager *interpreterConfigManager) updateConfiguration() error {
+	if configManager.informer == nil {
+		return errors.New("ResourceInterpreterCustomization informer not initialized")
```

Comment:
```suggestion
		return errors.New("informer manager is not configured")
```
I remember error messages usually do not start with a capital character.  (Do not know why lint missed it.)

Reply:
Lint may not have this check item; Goland will provide reminders during programming.